### DISCUSSION TITLE
behind-the-scenes-of-minimal-apis-5-generating-argument-expressions-for-minimal-apis-continuted/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#108] [Announcements]
# behind-the-scenes-of-minimal-apis-5-generating-argument-expressions-for-minimal-apis-continuted/

In this post I look at some more expressions that RequestDelegateFactory.CreateArgument() generates to bind minimal API handler parameters

https://andrewlock.net/behind-the-scenes-of-minimal-apis-5-generating-argument-expressions-for-minimal-apis-continuted/

**Comment by vince-nyanga** [comment]
This is a treasure trove of information. Thanks so much.
